bootstrap.py

```diff
@@ -455,6 +454,15 @@ def update_defaults(self, defaults):
         # 2. environmental variables
         config.update(dict(self.get_environ_vars()))
         # Then set the options with those values
+
+        def strtobool(string):
+            if string in ['y', 'yes', 't', 'true', 'on', '1']:
+                return True
+            elif string in ['n', 'no', 'f', 'false', 'off', '0']:
+                return False
+            else:
+                raise ValueError
```

Comment:
I think this should be an instance and not a class, and provide a failure reason (like "Unexpected string value %s for boolean, please select one of %s")